exercises/concept/gotta-snatch-em-all/.meta/Exemplar.elm

```diff
@@ -0,0 +1,11 @@
+module GottaSnatchEmAll exposing (someFunction, someOtherFunction)
```

Comment:
This needs doing!

Reply:
🙈 It's open in my editor, I forgot to save it.